<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>725</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Hear Demo Track Easy</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/images_ui/images/deejayDeFavicon.ico</normaloff>:/images_ui/images/deejayDeFavicon.ico</iconset>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QDockWidget" name="dockWidgetParameters">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>180</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>180</height>
       </size>
      </property>
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::LeftDockWidgetArea</set>
      </property>
      <property name="windowTitle">
       <string>Параметры выборки</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents"/>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="QDockWidget" name="dockWidgetDbView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>642</width>
        <height>542</height>
       </size>
      </property>
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::RightDockWidgetArea</set>
      </property>
      <property name="windowTitle">
       <string>База треков</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_13"/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDockWidget" name="dockWidgetSearch">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>110</height>
       </size>
      </property>
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::LeftDockWidgetArea</set>
      </property>
      <property name="windowTitle">
       <string>Поиск</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_7"/>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDockWidget" name="dockWidgetPlay">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>130</height>
       </size>
      </property>
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::LeftDockWidgetArea</set>
      </property>
      <property name="windowTitle">
       <string>Плеер</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_3"/>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QDockWidget" name="dockWidgetCover">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>130</height>
       </size>
      </property>
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::LeftDockWidgetArea</set>
      </property>
      <property name="windowTitle">
       <string>Обложка</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_5"/>
     </widget>
    </item>
   </layout>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>589</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="action_Fetch"/>
    <addaction name="action_Load"/>
    <addaction name="action_Cancel"/>
    <addaction name="separator"/>
    <addaction name="action_Clear"/>
    <addaction name="action_Remove"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_Info"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Параметры</string>
    </property>
    <addaction name="action_Preferences"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_ShowControl"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Fetch"/>
   <addaction name="action_Load"/>
   <addaction name="action_Search"/>
   <addaction name="action_Cancel"/>
   <addaction name="separator"/>
   <addaction name="action_Preferences"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_Clear">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images_ui/images/clear.png</normaloff>:/images_ui/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Очистить БД</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images_ui/images/delete.png</normaloff>:/images_ui/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить строку</string>
   </property>
   <property name="toolTip">
    <string>Удалить строку</string>
   </property>
  </action>
  <action name="action_Fetch">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images_ui/images/fetch.png</normaloff>:/images_ui/images/fetch.png</iconset>
   </property>
   <property name="text">
    <string>Выборка</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images_ui/images/download.png</normaloff>:/images_ui/images/download.png</iconset>
   </property>
   <property name="text">
    <string>Загрузка</string>
   </property>
  </action>
  <action name="action_Cancel">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images_ui/images/cancel.png</normaloff>:/images_ui/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images_ui/images/preferences.png</normaloff>:/images_ui/images/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="toolTip">
    <string>Настройки</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images_ui/images/exit.png</normaloff>:/images_ui/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_Info">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images_ui/images/information.png</normaloff>:/images_ui/images/information.png</iconset>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="action_ShowControl">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images_ui/images/controller.png</normaloff>:/images_ui/images/controller.png</iconset>
   </property>
   <property name="text">
    <string>Показать панель управления</string>
   </property>
  </action>
  <action name="action_Search">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images_ui/images/search.png</normaloff>:/images_ui/images/search.png</iconset>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
